android/ops/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.4.1)
+set(TARGET torchvision_ops)
+project(${TARGET} CXX)
+set(CMAKE_CXX_STANDARD 14)
+
+string(APPEND CMAKE_CXX_FLAGS " -DMOBILE")
+
+set(build_DIR ${CMAKE_SOURCE_DIR}/build)
+set(root_DIR ${CMAKE_CURRENT_LIST_DIR}/..)
+
+file(GLOB VISION_SRCS
+    ../../torchvision/csrc/cpu/*.h
+    ../../torchvision/csrc/cpu/*.cpp
+    ../../torchvision/csrc/*.h
+    ../../torchvision/csrc/*.cpp)
```

Comment:
FYI there are currently discussions for changing the structure of the library. If we do that, we will need to change this part.

Reply:
@IvanKobzarev We changed the structure. Have a look on the latest master.